BENCHMARKING.md

```diff
@@ -23,13 +23,12 @@ benchmarks](https://github.com/ctz/openssl-bench), which produce similar measure
 
 #### Building
 
-The benchmarks are implemented in the form of "example code" in `rustls/examples/internal/bench.rs`.
-Use `cargo build --profile=bench -p rustls --example bench` to obtain the corresponding binary (you can
-toggle conditionally compiled code with the `--no-default-features` and `--features` flags) or simply run below, which will build and run the benchmark.
+The benchmarks are implemented in the form of "example code" in `rustls-bench/src/main.rs`.
```

Comment:
one small nit: I think they're not technically examples anymore. Could probably drop the "example code" part here.

Reply:
Sure thing! You don't squash commits, so I amended the existing one, hope that works for you.

Reply:
That's perfect, thanks again.

Reply:
You're absolutely welcome, thanks for the very quick review!